examples/atlas-organization/organization-step-1/variables.tf

```
@@ -0,0 +1,52 @@
+variable "public_key" {
+  type        = string
+  description = "Public Programmatic API key to authenticate to Atlas"
+}
+variable "private_key" {
+  type        = string
+  description = "Private Programmatic API key to authenticate to Atlas"
+}
+variable "org_owner_id" {
+  type        = string
+  description = "MongoDB Organization Owner ID"
+}
+
+variable "project_name" {
+  type        = string
+  description = "The MongoDB Atlas Project Name"
+}
+variable "cluster_name" {
+  type        = string
+  description = "The MongoDB Atlas Cluster Name"
+}
+variable "cloud_provider" {
+  type        = string
+  description = "The cloud provider to use, must be AWS, GCP or AZURE"
+}
+variable "region" {
+  type        = string
+  description = "MongoDB Atlas Cluster Region, must be a region for the provider given"
+}
+variable "mongodbversion" {
+  type        = string
+  description = "The Major MongoDB Version"
+}
+variable "dbuser" {
+  type        = string
+  description = "MongoDB Atlas Database User Name"
+}
+variable "dbuser_password" {
+  type        = string
+  description = "MongoDB Atlas Database User Password"
+}
+variable "database_name" {
+  type        = string
+  description = "The database in the cluster to limit the database user to, the database does not have to exist yet"
+}
+variable "ip_address" {
+  type        = string
+  description = "The IP address that the cluster will be accessed from, can also be a CIDR range or AWS security group"
```

minor suggestion to make this clearer that it's a security feature and you won't be able to connect from elsewhere
```suggestion
  description = "The IP address that the cluster may be accessed from, can also be a CIDR range or AWS security group"
```

---

Removed IP address  variable as not relevant in this example